src/main/webapp/app/shared/metis/postings-reactions-bar/post-reactions-bar/post-reactions-bar.component.ts

```diff
@@ -19,4 +40,46 @@ export class PostReactionsBarComponent extends PostingsReactionsBarDirective<Pos
         reaction.post = this.posting;
         return reaction;
     }
+
+    /*
+    flips the value of the pinned property on a posting by invoking the metis service
+     */
```

Comment:
I would suggest for the comments here:
```suggestion
    /**
     * flips the value of the pinned property on a posting by invoking the metis service
     */
```

Reply:
I will change it